chapter_attention-mechanism/seq2seq-attention.md

```diff
@@ -3,20 +3,24 @@
 In this section, we add the attention mechanism to the sequence to sequence
 model introduced in :numref:`sec_seq2seq`
 to explicitly select state. :numref:`fig_s2s_attention` shows the model
-architecture for a decoding time step. As can be seen, the memory of the
-attention layer consists of the encoder outputs of each time step. During
-decoding, the decoder output from the previous time step is used as the query,
-the attention output is then fed into the decoder with the input to provide
-attentional context information.
+architecture for encoding and decoding at the time step $t$. Here, the memory of the
+attention layer consists of all the information that the encoder has 
+seen---the encoder outputs at each time step. 
+During the decoding, the decoder output from the previous time step $t-1$ is used as the query.
+While the output of the attention model is viewed as the context information, 
+which is then concatenated with the decode input $D_t$. Finally, we feed the concatenation into the decoder.
```

Comment:
decode input -> decoder input

Reply:
During the decoding, the decoder output from the previous time step $t-1$ is used as the query.
While the output of the attention model is viewed as the context information, 
which is then concatenated with the decode input $D_t$.

hard to understand